Upstream: Unknown
Reason: Allow usage of py3-praw in Alpine Linux

diff --git a/requirements.txt b/requirements.txt
index 5b27125..6aaf76e 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,7 +1,7 @@
 xmltodict<0.12.0; python_version == '3.3'
 xmltodict==0.12; python_version != '3.3'
 pytz
-praw>=4.0.0,<6.0.0
+praw>=4.0.0,<7.0.0
 geoip2<3.0.0
 ipaddress<2.0; python_version < '3.3'
 requests>=2.0.0,<3.0.0
